isis/src/base/apps/cathist/main.cpp

```diff
@@ -26,7 +26,8 @@ void IsisMain() {
   }
 
   // Extract history from file
-  History hist("IsisCube", fromfile.expanded());
+  Blob historyBlob("IsisCube", "History", fromfile.expanded());
```

Comment:
I am not a fan of having "IsisCube" and "History" hard coded in places outside of the History class. Not sure the best way around this, maybe some way to get an empty Blob for reading/writing History objects?

Reply:
Maybe? The only reason these are hard coded here is because we don't have a cube open. The "IsisCube" is also hard coded in the `readHistory` function on the cube and the `toBlob` method on the history object. They are both set in the associated `.h` files